libs/explore/feature/src/lib/expand-modal/expand-modal.component.spec.ts

```diff
@@ -1,40 +0,0 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
If you haven't saved this anywhere I recommend putting it in the pages main spec.ts so long. Same for the other modal